packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -106,20 +106,15 @@ const DomainPickerSuggestionItem: React.FC< Props > = ( {
 			? __( 'Default', __i18n_text_domain__ )
 			: __( 'Free', __i18n_text_domain__ );
 
-	const firstYearIncludedInPaid = isMobile
-		? __( 'Included in paid plans', __i18n_text_domain__ )
+	const paidIncludedDomainLabel = isMobile
+		? __( 'Included in annual plans', __i18n_text_domain__ )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Included with annual plans' )` *ES Score: 10*